medoo/base.py

```diff
@@ -144,4 +144,7 @@ def query(self, sql, commit=True, readonly=True):
             return True
         except Exception as ex:
             self.errors.append(str(ex))
-            raise type(ex)(str(ex) + ":\n" + self.sql)
+            if len(self.sql) <= 300:
+                raise type(ex)(str(ex) + ":\n" + self.sql)
+            else:
+                raise type(ex)(str(ex) + ":\n" + self.sql[:300] + "\nsql shown above is slimed, the full sql length is " + str(len(self.sql)))
```

Comment:
Try to break it into multiple lines,each within 80 chars.

Reply:
I just push a new commit, not sure if the pull has included the commit, or there are something else I need to do?